tests/test_dht_experts.py

```diff
@@ -59,11 +60,11 @@ def test_beam_search(
         {"expert." + ".".join([str(random.randint(0, dim - 1)) for dim in grid_dims]) for _ in range(total_experts)}
     )
     for batch_start in range(0, len(real_experts), batch_size):
+        dht_ = random.choice(dht)
```

Comment:
nit: Having both `dht` and `dht_` is confusing :) Let's rename them to `dht_instances` (a list) and `dht` (one instance) for clarity.